documentation/assemblies/assembly-upgrading.adoc

```diff
@@ -11,6 +11,10 @@ ifeval::["{cmdcli}" == "oc"]
 It is recommended to use the same method to upgrade that was used to initially install {ProductName}.
 endif::[]
 
+ifdef::ProductUpgrading[]
+When upgrading from {ProductName} 1.0 or 1.1.x, be sure to upgrade to the next available version before upgrading to {ProductName} 1.2.0.
```

Comment:
It is undesirable to refer to downstream version numbers within upstream documentation.  Can this be factored out using variables?

What do you think to the alternative words "be sure to first upgrade to the latest micro release of your current release before upgrading  {ProductName} {ProductVersion}

Reply:
@k-wall agreed about the release numbers. Let me think on which is the best approach (using three variables or rewording); leaning toward the rewording at the moment